tensorflow/python/keras/losses.py

```diff
@@ -1683,10 +1683,11 @@ def cosine_similarity(y_true, y_pred, axis=-1):
 
   Note that it is a number between -1 and 1. When it is a negative number between
   -1 and 0, 0 indicates orthogonality and values closer to -1 indicate greater
-  similarity. This makes it usable as a loss function in a setting where you try
-  to maximize the proximity between predictions and targets. If either `y_true`
-  or `y_pred` is a zero vector, cosine similarity will be 0 regardless of the
-  proximity between predictions and targets.
+  similarity. The values closer to 1 indicate greater dissimilarity. This makes
```

Comment:
This statement isn't right??

Reply:
Why?

If the data is `[1, 1]` and `[1, 1]` the vectors are completely similar, the cosine similarity is -1 (dot product is 1, we have a - in front).

If the data is `[1, 1]` and `[-1, -1]` the vectors are opposite each other, the dot product is -1, but due to the `-` in front the metric becomes 1.

If the data is `[0, 1]` and `[1, 0]`, the vectors are orthogonal, and both the dot product as well as the metric value are 0.

Reply:
@mihaimaruseac I'm sorry, I'm a little bit confused.
In the previous comment, I meant doesn't that statement(`The values closer to 1 indicate greater dissimilarity.`) include a case where output gets closer to 1?

Reply:
I meant that you should include the case where output gets closer to 1 in the doctest example below. The docstring header is good.